josm-presets/de-signals-bostrab.xml

```diff
@@ -1709,5 +1741,39 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 					/>
 			</item>
 		</group>
+		<group name="Hannover (ÜSTRA)">
+			<item name="Short Speed Limit Sign (G 5)" de.name="Langsamfahrstelle (G 5)" type="node">
```

Comment:
Could you please add an icon as well? Please choose a size equal to the G 1a signal. You can use the SVG you published at [on the OSM wiki](https://wiki.openstreetmap.org/wiki/File:BOStrab_G5.svg). Btw, please add a license to the file on the wiki, I recommend CC-0.